src/activities/SetGlobalProperty.ts

```diff
@@ -0,0 +1,64 @@
+import type { IActivityHandler } from "@geocortex/workflow/runtime";
+import config from "@arcgis/core/config";
+
+interface SetGlobalPropertyInputs {
+    /**
+     * @description The name of the global property to modify.
+     * @required
+     * @noExpressions
+     */
+    name:
+        | "apiKey"
+        | "fontsUrl"
+        | "geoRSSServiceUrl"
+        | "geometryServiceUrl"
+        | "kmlServiceUrl"
+        | "routeServiceUrl";
+
+    /**
+     * @description The value to apply to the property.
+     * @required
+     */
+    value: string;
+}
+
+interface SetGlobalPropertyOutputs {}
+
+/**
+ * @category ArcGIS Maps SDK for JavaScript
+ * @clientOnly
+ * @description Configure global properties of the ArcGIS Maps SDK for JavaScript library.
+ * @helpUrl https://developers.arcgis.com/javascript/latest/api-reference/esri-config.html
+ * @supportedApps EXB, GWV
+ */
+export default class SetGlobalProperty implements IActivityHandler {
+    execute(inputs: SetGlobalPropertyInputs): SetGlobalPropertyOutputs {
+        const { name, value } = inputs;
+        if (!name) {
+            throw new Error("name is required");
+        }
+
+        // Ensure the name input only includes a subset of config properties
+        // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
+        const prop = name as keyof Pick<
+            config,
+            SetGlobalPropertyInputs["name"]
+        >;
+
+        switch (prop) {
+            case "apiKey":
+            case "fontsUrl":
+            case "kmlServiceUrl":
```

Comment:
Should be after `"geometryServiceUrl"` for alphabetical order.

Reply:
I would tend to agree. I went with this because it is the alphabetical order provided by VS Code's code completion suggestions. I personally don't really care so long as there is a sensible reason for the order, but this did raise my eyebrow when I wrote it. I'll change it.

Reply:
Done

Reply:
Sorry, I had meant the `kmlServiceUrl` should have come after the "g" entries.

Reply:
Ah, shoot. Fixed.